tests/backup/backup_test.go

```diff
@@ -6408,6 +6408,195 @@ var _ = Describe("{IssueMultipleDeletesForSharedBackup}", func() {
 	})
 })
 
+// Share backup with created with same name between two users
+var _ = Describe("{SwapShareBackup}", func() {
+
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/82940
+	numberOfUsers := 2
+	users := make([]string, 0)
+	userBackupLocationMapping := map[string]string{}
+	var backupUIDList []string
+	var backupName string
+	var contexts []*scheduler.Context
+	var backupLocationUID string
+	var appContexts []*scheduler.Context
+	var bkpNamespaces []string
+	var clusterUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var credNames []string
+	var cloudCredUID string
+	bkpNamespaces = make([]string, 0)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("SwapShareBackup",
+			"Share backup with same name between two users", nil, 82940)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Share the backup with same name", func() {
+		providers := getProviders()
+		Step("Create Users", func() {
+			log.InfoD("Creating %d users", numberOfUsers)
+			var wg sync.WaitGroup
+			for i := 1; i <= numberOfUsers; i++ {
+				userName := fmt.Sprintf("testuser%v", i)
+				firstName := fmt.Sprintf("FirstName%v", i)
+				lastName := fmt.Sprintf("LastName%v", i)
+				email := fmt.Sprintf("testuser%v@cnbu.com", i)
+				wg.Add(1)
+				go func(userName, firstName, lastName, email string) {
+					defer wg.Done()
```

Comment:
add defer ginkgo.Recover() as it will get into panic if there is an error in go routine
```defer ginkgo.GinkgoRecover()```

Reply:
done